tests/utils/test_terms.py

```diff
@@ -0,0 +1,89 @@
+"""Provides unit tests for the utils.terms module"""
+
+# Standard
+import re
+
+# Third-party
+import pytest
+import rdflib.graph
+import rdflib.term
+
+# Local
+from abis_mapping import utils
+
+# Typing
+from typing import Iterable
+
+
+a = rdflib.RDF.type
+
+
+@pytest.fixture(scope="module")
+def patch_rdflib_bnode() -> Iterable[None]:
+    """Expected overriding/patching of rdflib.BNode will be done at the start of a module"""
+    rdflib.BNode = utils.terms.BNodeAlwaysUUID4  # type: ignore[misc]
+    rdflib.term.BNode = utils.terms.BNodeAlwaysUUID4  # type: ignore[misc]
+
+    yield None
+
+    rdflib.BNode = rdflib.BNode  # type: ignore[misc]
+    rdflib.term.BNode = rdflib.term.BNode  # type: ignore[misc]
```

Comment:
I think this may not be un-patching as intended... `rdflib.BNode` is now the custom class, so this is a no-op

Just use the regular patcher e.g. `mocker.patch.object(rdflib, "BNode", new=utils.terms.BNodeAlwaysUUID4`

Reply:
fixed.